editor/grida-canvas/reducers/event-target.reducer.ts

```
@@ -0,0 +1,1410 @@
+      break;
+    case "zoom":
+      // TODO: also support zoom out (with alt key modifier) - needs to be handled separately
+      draft.transform = cmath.transform.scale(
+        2,
+        draft.transform,
+        // map the cursor position back to surface space
+        cmath.vector2.transform(draft.pointer.position, draft.transform)
+      );
+      break;
+    case "insert":
+      const parent = __get_insertion_target(draft);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)